### DISCUSSION TITLE
Let the player make half time or daycore mods slower than 0.5x

### DISCUSSION THREAD

**Discussion by Hashtop** [ppy/osu#22680] [Ideas]
I think this would help a lot when practicing maps, since you can make it like 0.1x and then go up from there on .1x steps once you master it. It's just that, for some maps, 0.5x is still too fast to actually get consistent and make the accuracy better.

I honestly don't think this would be too hard to implement... but I may be wrong lol

**Comment by bdach** [comment]
Hard? No. But it'll probably sound terrible due to having to stretch sound samples out to keep constant pitch (unless https://github.com/ppy/osu/discussions/13125 is accepted, and even then that only applies to daycore).

**Reply by mk56-spn** [reply]
Yeah also against this for audio quality reasons. Not to mention it further messes with animations which already look crappy at 0.5x and 2x. It's a race to the bottom and with little reason since the shear amount of mods allows much more flexibility per beatmap than could ever be needed

**Reply by Hashtop** [reply]
Yes, streching audio files makes it sound worse the more you stretch. But, I still think it would be nice to have the option to do so...

**Reply by mk56-spn** [reply]
Well. The way I see it the limit has to be put somewhere. Music is a core part of the OSU experience and if 3x is allowed then people will obviously want it to submit and I think a large part of the community would not enjoy every 2 star top play to be an ear grating mess with horridly stretched / squashed animations

**Reply by mk56-spn** [reply]
Sorry. 0.33x*. since this is half time

**Reply by Hashtop** [reply]
Ok, fair enough.
